app.js

```diff
@@ -113,6 +113,8 @@ global.Ladders = require(Config.remoteladder ? './ladders-remote.js' : './ladder
 
 global.Users = require('./users.js');
 
+global.Punishments = require('./punishments.js');
```

Comment:
Problem with that would be that they're used in both users.js and
commands.js. I guess I could add a Users.Prototype.ban() that just calls
Punishments.ban(this, etc, etc) but it's fine enough with the global I
think.